test/integration/e2eissues_int_test.go

```diff
@@ -0,0 +1,48 @@
+package integration
+
+import (
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+)
+
+/*
+Tests in this file are failing due to bugs in the test framework. They are skipped so as not to break our CI.
+The intend is to collect e2e bugs here so that we can test that they are fixed once we refactor the framework.
+*/
+
+func TestMultipleSends(t *testing.T) {
+	t.Skip("")
```

Comment:
Should we have a skip reason? Or is the comment above enough?

Reply:
I feel like the comment above is enough, since the intend of this entire test suite is to be skipped until we're ready to run them again.